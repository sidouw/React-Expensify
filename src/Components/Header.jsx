import React from 'react' 
import {NavLink} from 'react-router-dom'
const Header = ()=>(
    <header>
    <h1>Epensify</h1>
    <NavLink to="/" activeClassName ="is-active" exact ={true}>Dashboard</NavLink>
    <NavLink to="/AddExpense" activeClassName ="is-active">Create expense</NavLink>
    <NavLink to="/Help" activeClassName ="is-active">Help</NavLink>
    </header>
)

export default Header