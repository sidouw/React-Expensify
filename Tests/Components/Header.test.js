import React from 'react'
import {shallow} from 'enzyme'
import Header from '../../src/Components/Header.jsx'

test('should render Header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });