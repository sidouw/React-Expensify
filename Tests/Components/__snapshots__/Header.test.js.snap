// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Epensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/AddExpense"
  >
    Create expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/Help"
  >
    Help
  </NavLink>
</header>
`;
